server/src/daemon/secret.rs

```diff
@@ -0,0 +1,41 @@
+use oo7::{crypto, Key};
+
+use super::session::Session;
+
+pub struct Secret {
+    session: Session,
+    parameters: Vec<u8>,
+    value: Vec<u8>,
+    content_type: String,
```

Comment:
Except the `value` field others are causing the clippy warnings. What's the best approach to fix them?

```
warning: fields `session`, `parameters`, and `content_type` are never read
 --> server/src/daemon/secret.rs:6:5
  |
5 | pub struct Secret {
  |            ------ fields in this struct
6 |     session: Session,
  |     ^^^^^^^
7 |     parameters: Vec<u8>,
  |     ^^^^^^^^^^
8 |     value: Vec<u8>,
9 |     content_type: String,
  |     ^^^^^^^^^^^^
  |
  = note: `#[warn(dead_code)]` on by default
```